"error","count"
"{{getv4,getv4},{get,connection_closing}}","108"
"{{getv4,getv4},{http_error,[53,48,48]}}","24517"
"{{putv4,putv4},{http_error,[52,48,48]}}","1"
"{{putv4,putv4},{http_error,[53,48,48]}}","130"
